content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md

```diff
@@ -129,32 +129,31 @@ You must create the projects as shown in the table below in advance. Make sure y
    
            REGISTRY = 'docker.io'
            DOCKERHUB_NAMESPACE = 'your Docker Hub account ID'
-           APP_NAME = 'devops-java-sample'
+           APP_NAME = 'devops-maven-sample'
            SONAR_CREDENTIAL_ID = 'sonar-token'
            TAG_NAME = "SNAPSHOT-$BRANCH_NAME-$BUILD_NUMBER"
        }
      stages {
        stage('checkout') {
          steps {
            container('maven') {
-             git branch: 'master', url: 'https://github.com/kubesphere/devops-java-sample.git'
+             git branch: 'master', url: 'https://github.com/kubesphere/devops-maven-sample.git'
            }
          }
        }
        stage('unit test') {
          steps {
            container('maven') {
-             sh 'mvn clean -o -gs `pwd`/configuration/settings.xml test'
+             sh 'mvn clean test'
            }
-   
          }
        }
        stage('sonarqube analysis') {
          steps {
            container('maven') {
              withCredentials([string(credentialsId: "$SONAR_CREDENTIAL_ID", variable: 'SONAR_TOKEN')]) {
                withSonarQubeEnv('sonar') {
-                 sh "mvn sonar:sonar -o -gs `pwd`/configuration/settings.xml -Dsonar.login=$SONAR_TOKEN"
+                 sh "mvn sonar:sonar -Dsonar.login=$SONAR_TOKEN"
                }
    
```

Comment:
Is the format here right in the EN document?

![Xnip2021-11-16_14-46-42](https://user-images.githubusercontent.com/75110798/141935186-3261558d-1dd3-4405-a45b-53b848a80928.png)

Reply:
It is ok to have the extra whitespace.